src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -426,7 +437,7 @@
                       </padding>
                     </HBox>
                     <HBox alignment="CENTER_LEFT" spacing="4.0">
-                      <Label text="Percentage Threshold *" prefWidth="196.0"/>
+                      <Label prefWidth="196.0" text="Percentage Threshold"/>
```

Comment:
Is your eventual plan to hide this box entirely instead of disabling it? It's a confusing UX to see a field that's labeled required but also disabled, especially because the disabled state is visually almost indistinguishable from the enabled state.

Reply:
Regarding "bottoms-up using percentage threshold": yeah, ideally we could just hide "Number of Winners" when this mode is selected, as it doesn't make sense for the user to configure that field. (And the fact that it's set to zero is just an internal implementation detail.)

Reply:
Yeah, I do think ultimately we'll go with hiding it, but it likely won't be in this iteration. It's pretty clear when elements are disabled in Windows, but it sounds like it's not in MacOS? I need to check in Linux too before finalizing all this GUI stuff.

Reply:
The disabled checkboxes are pretty clear. But the text inputs are super subtle. It would help a lot if the text input labels were grayed out in the same way that the checkbox labels are.

![image](https://user-images.githubusercontent.com/352413/91257928-ed551500-e71f-11ea-8d97-fe23555da9bb.png)